wagtail/admin/checks.py

```diff
@@ -237,3 +237,41 @@ def file_overwrite_check(app_configs, **kwargs):
         )
 
     return errors
+
+
+@register("datetime_format")
+def datetime_format_check(app_configs, **kwargs):
```

Comment:
```suggestion
def datetime_format_check(app_configs, **kwargs):
    """
    If L10N is enabled, check if WAGTAIL_* formats are compatible with Django input formats.
    See https://docs.djangoproject.com/en/stable/topics/i18n/formatting/#creating-custom-format-files
    See https://docs.wagtail.org/en/stable/reference/settings.html#wagtail-date-format-wagtail-datetime-format-wagtail-time-format
    """
```

Let's move the comment below into the docstring and add some context.